### PR TITLE
Setup associations

### PR DESCRIPTION
**Pull request by mnaibei**
- Added new classes
- Set up associations between the classes

### CODE REVIEW

File: classroom.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = [] # Initialize an empty array to store the students associated with the classroom
+  end
+end
```
Comment (julie-ify) [inline classroom.rb]: - [ ] The classroom should have a method for the relationship with the student class. Kindly add it.

  ```
  def add_student(student)
    @students << student
    student.classroom = self
  end
  ```

File: book.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    # @title and @author instance variables
+    @title = title
+    @author = author
+  end
+end
```
Comment (julie-ify) [inline book.rb]: - [ ] The Book class should have `@rentals` array in the constructor and add it to the `attr_accessor`:

  ```
  attr_accessor :title, :author, :rentals

  def initialize(title, author)
    @title = title
    @author = author
    @rentals = []
  end
  ```

- [ ] As you have added the `add_rental` method in the Person class kindly do the same in the Book class. Be sure to take note of the order of the arguments of the rental object.

  ```
  def add_rental(date, person)
    Rental.new(date, self, person)
  end
  ```

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @mnaibei,

Good job so far!

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — APPROVED** [review]
### Hi @mnaibei, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**